### PR TITLE
First 3 sections portfolio project

### PR DESCRIPTION
**Pull request by mbdesigns1989**
Portfolio Project
First 3 sections portfolio project
add markup 
renamed repository
add descriptive commits

### CODE REVIEW

**Comment by mbdesigns1989** [comment]
If there are any errors please send me video links to fix them.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,500;1,400&display=swap" rel="stylesheet">
+
+  <title>My Portfolio</title>
+</head>
+<body>
+<header>
+  <nav class="header">
+    <a href="#" class="logo  nav-link"><h1 >My Logo</h1></a>
```
Comment (Lameck1) [inline index.html]: - [ ] We only need one `<h1>` on the page for SEO purposes. Please consider refactoring this to a `<span>` for example.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  margin-right: 5%;
+}
+
+.information {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
```
Comment (Lameck1) [inline styles.css]: - [ ] [OPTIONAL] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been used several times in the CSS. Please refactor it to a class and apply the class in the HTML file to elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to HTML element that needs this behavior:

```HTML
  <section class="d-flex">
```

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+.icon-links {
+  display: flex;
+  justify-content: flex-start;
+  list-style: none;
+  margin-left: 0;
+}
+
+.icon-links li {
+  margin-right: 5%;
+}
```
Comment (Lameck1) [inline styles.css]: - [ ] Alignment of this section is also off as compared to the provided Figma design. Take a look 👇🏼. Consider working on it.

Implementation            |  Actual Design
:-------------------------:|:-------------------------:
social_imp  |  social

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi @mbdesigns1989, 

What you've done so far is just awesome 👌🏼.

We still, however, have to iron out some minor issues before proceed with the next tasks.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav-link {
+  margin: 2%;
+  color: #6070ff;
+  margin-top: 10%;
+}
```
Comment (Lameck1) [inline styles.css]: - [ ] Please ensure you have matched the design for the header section.

Implementation             |  Figma Design
:-------------------------:|:-------------------------:
nav_implementation  |  nav_design

File: index.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,500;1,400&display=swap" rel="stylesheet">
+
+  <title>My Portfolio</title>
+</head>
+<body>
+<header>
+  <nav class="d-flex header">
+    <a href="#" class="logo  nav-link"><span>My Logo</span></a>
+    <span class="material-icons nav-link" >menu</span>
+  </nav>
```
Comment (Emmyn5600) [inline index.html:19]: - [ ]  Please ensure you have matched the design for the header section. For more details check the screenshot image attached by the previous reviewer.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+  <section class="article">
+    <h1 class="d-flex information">I'm Mohamed Badr <br> glad to see <br> you!</h1>
+    <p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! <br>If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <h2 class="contact">LET'S CONNECT</h2>
+    <ul class="d-flex  icon-links">
+      <li><a href="https://github.com/mbdesigns1989"><img src="./img/github.png" alt=""></a></li>
+      <li><a href="https://www.linkedin.com/in/mohamed-badr-27b26a212/"><img src="./img/linkedin.png" alt=""></a></li>
+      <li><a href="https://twitter.com/mohamed66083152"><img src="./img/twitter.png" alt=""></a></li>
+    </ul>
```
Comment (Emmyn5600) [inline index.html]: - [ ]  Alignment of this section is also off as compared to the provided Figma design. Take a look 👇🏼. Consider working on it. For more details again check the screenshot image attached by the previous reviewer.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: Changes Requested :recycle:

Hi @mbdesigns1989 

Good job so far!:clap::clap:
You have done well on making the changes requested by the previous code reviewer:clap:, However, there are some issues that you still need to work on to go to the next project but you are almost there!:muscle:

##  Required Changes:writing_hand:

**_Check the comments under the review._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by mbdesigns1989** [comment]
please approve my project as this is the last time to review and I worked so hard on this project, thanks.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi MB Taha, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 👍.
Congratulations! 🎉


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
__As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have no more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).__

**Pull request merged**